packages/destination-actions/src/destinations/fullstory/index.ts

```diff
@@ -41,6 +41,9 @@ const destination: DestinationDefinition<Settings> = {
   },
 
   onDelete: async (request, { settings, payload }) => {
+    if (payload.userId === null || payload.userId === undefined) {
```

Comment:
Short circuiting this return allowed me to standardize on `userId: string` instead of `userId: ID` in `request-params.ts` which felt more correct (our API URLs expected a string) and also eliminated the need to mirror the `ID` type that isn't exported from `@segment/actions-core`.